contracts/YieldStrategyLever.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.14;
+
+import "./YieldLeverBase.sol";
+
+interface IStrategy {
+    function mint(address to) external returns (uint256 minted);
+
+    function burn(address to) external returns (uint256 withdrawal);
+
+    function burnForBase(address to) external returns (uint256 withdrawal);
+
+    function pool() external returns (IPool pool);
+}
+
+contract YieldStrategyLever is YieldLeverBase {
```

Comment:
This contract needs some nice natspec on top explaining how it works. Ascii diagrams welcomed.